Dockerfile

```diff
@@ -0,0 +1,25 @@
+ARG PYTHON_VERSION=3.12
+FROM python:${PYTHON_VERSION}-slim AS base
+LABEL maintainer="vlados.soc@gmail.com"
+
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . .
+
+RUN mkdir -p /files/media

```

Comment:
The RUN command for creating directories could be combined with the chown and chmod commands to reduce the number of layers in the Docker image. Consider combining these commands into a single RUN statement.